src/Controllers/ElementalAreaController.php

```diff
@@ -139,7 +139,7 @@ public function save(array $data, Form $form): HTTPResponse
         $dataWithoutNamespaces = static::removeNamespacesFromFields($data, $element->ID);
 
         // Update and write the data object which will trigger model validation
-        $element->update($dataWithoutNamespaces);
+        $element->updateFromFormData($dataWithoutNamespaces);
```

Comment:
I've added this code change in from a [different PR](https://github.com/silverstripe/silverstripe-elemental/pull/1180/files#diff-942115e4b8f6030e4d72ebc2b3a0772ec65e5dfcd08fbd0e677c70d1231daf28R142) that's blocked on this one which is required to get the inline-block-validation.feature behat tests to pass